### PR TITLE
Removed DOM Errors and Added initial commit for join vrikshit component

### PR DESCRIPTION
**Pull request by shankarsingh482**
image

image

### CODE REVIEW

File: frontend/src/components/Pages/JoinVrikshit/joinVrikshit.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    return (
+      <div className="container description_Form">
+        <div className="row">
+          <div className="col-md-12">
+            <h1 className="volunteer_headline">VOLUNTEER WITH US</h1>
+            <iframe
+              src="https://docs.google.com/forms/d/e/1FAIpQLScPBGeteil_ggg3pHZaOou7KNEMxf04WgbPKoV43WksmJFzaQ/viewform?embedded=true"
+              width="1200"
+              height="675"
+              frameborder="0"
```
Comment (avantikabharti) [inline frontend/src/components/Pages/JoinVrikshit/joinVrikshit.jsx]: this width/height will make page unresponsive.

Comment (shankarsingh482) [reply frontend/src/components/Pages/JoinVrikshit/joinVrikshit.jsx]: Right , removing the width and height

Comment (shankarsingh482) [reply frontend/src/components/Pages/JoinVrikshit/joinVrikshit.jsx]: @itsnamangoyal  - From next time , please cut different branch name it as **feature-css**

Comment (itsnamangoyal) [reply frontend/src/components/Pages/JoinVrikshit/joinVrikshit.jsx]: alright

File: frontend/src/App.css [hunk 2]
```diff
@@ -12,17 +12,16 @@
...
+    padding-top: 15%;
   }
 }
 
 @media only screen and (max-width: 480px) {
   .all-Pages-Common {
-    padding-top: 36%;
+    padding-top: 17%;
   }
 }
```
Comment (avantikabharti) [inline frontend/src/App.css:23]: image
I see a regression around this class, there is space between header and page content. @itsnamangoyal  please check that.

**Pull request merged**